docs/customizations/custom-css.md

```diff
@@ -44,6 +44,32 @@ Finally, select the classname of the element you want to customize and add it to
 
 ---
 
+### Static Classes
+
+Homarr has a few static classes, which you can use for your CSS.
```

Comment:
```suggestion
By default, many class names will be made out of a random value, also known as a hash.
These are used internally by Mantine and can make writing custom CSS quite difficult.
The biggest issue with these hashes is, that they will be randomized each build - which means, if you update Homarr, they will change and your CSS might no longer work.
For this reason, we recommend you to avoid the random hash class-names and implemented static ones, for the most important objects on your dashboard, that are easier to use and don't change each build.
```